src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -834,7 +841,22 @@ public int getSize() {
      *             the io exception
      */
     public PagedIterable<GHUser> listCollaborators() throws IOException {
-        return listUsers("collaborators");
+        return listCollaborators(CollaboratorAffiliation.ALL);
+    }
+
+    /**
+     * Lists up the collaborators on this repository.
+     *
+     * @param affiliation
+     *            Filter users by affiliation
+     * @return Users paged iterable
+     * @throws IOException
+     *             the io exception
+     */
+    public PagedIterable<GHUser> listCollaborators(CollaboratorAffiliation affiliation) throws IOException {
+        Map<String, Object> args = new HashMap<>();
+        args.put("affiliation", affiliation.toString().toLowerCase());
```

Comment:
```suggestion
        args.with("affiliation", affiliation);
```

Lowercasing is done automatically for enums [inside the request builder](https://github.com/hub4j/github-api/blob/208904b6348e6e386c0f74d42ff1f2ed9a5e8aa5/src/main/java/org/kohsuke/github/GitHubRequest.java#L491).

Reply:
Oh! Lovely. I'll remove the superfluous calls.

Reply:
Oh, wait, sorry.  This doesn't work.  The request builder `with(String, Enum)` is what does the toLowercase for you.   You're passing a Map.  Maybe switch to passing  a request builder?  It's okay if you want to keep this as is.